src/controllers/plugins/LuaUtilities.hpp

```diff
@@ -186,6 +291,55 @@ StackIdx pushEnumTable(lua_State *L)
     return out;
 }
 
+// Represents a Lua function on the stack
+template <typename ReturnType, typename... Args>
+class CallbackFunction
+{
+    StackIdx stackidx;
+    lua_State *L;
+
+public:
+    CallbackFunction(lua_State *L, StackIdx stackid)
```

Comment:
naming/casing of member `stackidx` & paramter `stackid` should be consistent, probably `stackIdx` or `stackIndex`